spanner/integration_test.go

```diff
@@ -3697,6 +3783,15 @@ func skipEmulatorTest(t *testing.T) {
 	}
 }
 
+func skipUnsupportedPGTest(t *testing.T) {
+	if testDialect == adminpb.DatabaseDialect_POSTGRESQL {
+		skipEmulatorTest(t)
```

Comment:
I don't think we need to add this. If we want to skip a test on the emulator, we can directly call `skipEmulatorTest`.